### ISSUE TITLE
Application permissions for unsubscribeByMail

### ISSUE THREAD

**Issue by rboyatt** [microsoftgraph/microsoft-graph-docs#10165]
In the beta documentation only, the permissions table suggests that this supports application permission types.  

The table lists 'Group.ReadWrite.All' against 'Application' permissions but the documentation does not given any indication of how this is possible and this type of access is not available in the v1.0 documentation.  

Either the permissions table is incorrect, or additional documentation is needed to demonstrate how this API can be used with application permissions.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c9b1e398-bb89-af3e-fed2-2d3a141ed1c2
* Version Independent ID: 58c61256-e252-7ff8-9ae1-9e3687df117f
* Content: [group: unsubscribeByMail - Microsoft Graph beta](https://docs.microsoft.com/en-us/graph/api/group-unsubscribebymail?view=graph-rest-beta&tabs=http)
* Content Source: [api-reference/beta/api/group-unsubscribebymail.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/beta/api/group-unsubscribebymail.md)
* Product: **groups**
* Technology: **microsoft-graph**
* GitHub Login: @yyuank
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by davidmu1** [comment]
@rboyatt - Can you please clarify what you are trying to do? Are you trying to use the beta API and having issues? Or, are you wanting to use the API in v1.0 and are wondering why the permission isn't available there? No change identified for the page. Closing the issue until more information provided.

**Comment by rboyatt** [comment]
@davidmu1 I'd been looking for ways to unsubscribe users from emails about events associated with a group using application, rather than delegated, permissions.

I'd looked at the v1.0 API (https://docs.microsoft.com/en-us/graph/api/group-unsubscribebymail?view=graph-rest-1.0&tabs=http) and noted this only supports delegated permissions.

When I looked at the beta API (https://docs.microsoft.com/en-us/graph/api/group-unsubscribebymail?view=graph-rest-beta&tabs=http) I noticed that the table under 'Permissions' claims the API supports _both_ delegated and application permissions.

![image](https://user-images.githubusercontent.com/900659/97482989-c9e05d80-194e-11eb-8365-ad718a3e047f.png)

However the beta documentation doesn't show how this can be used in an application context.  No request details or examples are given.  So I think the documentation either needs to explain how this API can be used as an 'application' (as well as 'delegated'), or the table in the beta documentation incorrectly claims the API can be used in this way.
